collie/model/hybrid_matrix_factorization.py

```diff
@@ -2,7 +2,7 @@
 import os
 from pathlib import Path
 from typing import Callable, Dict, List, Optional, Union
-from collections.abc import Iterable
+from typing import Iterable
```

Comment:
Can we combine this import with the line above this?

Reply:
this is what I get for making changes late at night, can't believe I didn't see `typing` one line up 🤦

Reply:
👍